it/uk/gov/hmrc/minorentityidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -138,7 +138,7 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with A
 
       lazy val result = post("/minor-entity-identification/api/trusts-journey", testJourneyConfigJson)
 
-      (result.json \ "journeyStartUrl").as[String] must include(testContinueUrl + s"?journeyId=$testJourneyId")
+      (result.json \ "journeyStartUrl").as[String] must include(trustControllerRoutes.TrustUtrController.show(testJourneyId).url)
```

Comment:
Could we also have a test to check that when the feature switch is enabled it goes to this utr page but if it's disabled it goes to the continue url